Kernel/Arch/aarch64/Time/PL031.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2025, the SerenityOS developers
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/StdLibExtras.h>
+#include <AK/Time.h>
+#include <Kernel/Memory/TypedMapping.h>
+
+// https://developer.arm.com/documentation/ddi0224/c/?lang=en
+
+namespace Kernel {
+
+class PL031 {
+public:
+    struct RTCRegisters {
```

Comment:
We usually use more descriptive names (typically similar to the register description/long name from the datasheet) for registers instead of the short abbreviated ones.

You can take some inspiration from the datasheet register description and change the struct to maybe something like this (maybe also take a look at the GIC driver):
```c++
struct RTCRegisters {
    u32 data;  // RTCDR
    u32 match; // RTCMR
    [...]
};
```
(adding a comment with the "offical" register name is nice so you can find it more easily in the datasheet.)

Granted, `data` isn't very descriptive either, but at least it's better than `RTCDR`.